Monika-After-Story/MonikaModDev
# Echo's short hairs

Sprite pack that includes short hair versions of down and ponytail. Created by Echo.

## Testing

- verify ponytail allows ribbons (and they look good) and down does not.

- verify all available accessories look good with both hairs

- verify hair looks good
